backend/uclapi/timetable/management/commands/update_gencache.py

```diff
@@ -345,6 +354,24 @@ def handle(self, *args, **options):
             # to be run again in the future.
             self._redis.delete(cache_running_key)
 
+            incident_name = get_incident_name("Gencache")
+            if incident_name:
+                try:
+                    delete_incident(incident_name)
+                except CachetException as cachet_error:
+                    print(f"Failed to delete cachet incident. "
+                          f"Reason: {repr(cachet_error)}")
+                except Exception as cachet_error:
+                    print(f"Unexpected: Failed to delete cachet incident. "
+                          f"Reason: {repr(cachet_error)}")
+            else:
+                print("Could not find appropriate incident in Cachet!")
+
+            if settings.UCLAPI_DOMAIN_CURRENT == "staging.ninja":
+                delete_incident("Gencache-Staging")
+            elif settings.UCLAPI_DOMAIN_CURRENT == "uclapi.com":
+                delete_incident("Gencache-Prod")
+
             call_command('trigger_webhooks')
         except Exception as gencache_error:
```

Comment:
The same comment in `feed_occupeye_cache.py` regarding use of `try/except/else` applies here as well. Unfortunately, unlike in the `try` for occupeye, there's going to have be a lot of code in the `try` block AFAICT, but at least we can move the incident handling to the `else`.